war/src/main/webapp/css/style.css

```diff
@@ -1891,3 +1891,28 @@ body.no-sticker #bottom-sticker {
+
+.remodal {
```

Comment:
[RU About bem](https://www.youtube.com/watch?v=LznY9VUK-h8&index=11&list=PLLkvpHo_HuBMU_fM4v-VS5VbUi9QuKyDR)

You should describe your container with help of bem methodology. Then simply use remodal inside of your container.